src/lib/AnnotationControls.js

```diff
@@ -0,0 +1,63 @@
+import noop from 'lodash/noop';
+import { ICON_REGION_COMMENT } from './icons/icons';
+import Controls, { CLASS_BOX_CONTROLS_GROUP_BUTTON } from './Controls';
+
+const CLASS_ANNOTATIONS_GROUP = 'bp-annotations-group';
+const CLASS_REGION_COMMENT_BUTTON = 'bp-region-comment-btn';
+const CLASS_BUTTON_ACTIVE = 'active';
+
+class AnnotationControls {
```

Comment:
Since this is a new component, can it be written in TypeScript?

Reply:
> Since this is a new component, can it be written in TypeScript?

Will do